ext/sockets/sockets.c

```diff
@@ -76,8 +76,12 @@
 
 ZEND_DECLARE_MODULE_GLOBALS(sockets)
 
-#ifndef SUN_LEN
-#define SUN_LEN(su) (sizeof(*(su)) - sizeof((su)->sun_path) + strlen((su)->sun_path))
```

Comment:
We still need this for Windows unconditionally.

Reply:
I think we need it everywhere, where `SUN_LEN` isn't already defined.